airgun/views/computeresource.py

```diff
@@ -122,27 +127,43 @@ def is_displayed(self):
 class ResourceProviderDetailView(BaseLoggedInView):
```

Comment:
Probably `class ResourceProviderEditView(BaseLoggedInView):` is more should be named as `class ResourceProviderCreateView(BaseLoggedInView):`

and current one `ResourceProviderDetailView` probably should be names `ResourceProviderEditView`